app/models/application.rb

```diff
@@ -0,0 +1,171 @@
+# == Schema Information
+#
+# Table name: applications
+#
+#  id                     :integer          not null, primary key
+#  user_id                :integer
+#  track                  :string
+#  over_18                :boolean
+#  previous_attendance    :string
+#  programming_experience :text
+#  reason                 :text
+#  tshirt_size            :string
+#  admin_status           :string
+#  user_status            :string
+#  gender                 :string
+#  dietary_requirements   :json
+#  extra_info             :text
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+
+class Application < ActiveRecord::Base
+  validates :track, :previous_attendance, :programming_experience,
+            :reason, :tshirt_size, :gender, presence: true
+  validates_inclusion_of :over_18, in: [true, false]
+
+
+  # relations
+  belongs_to :user
+  acts_as_commentable
+  has_many :votes
+
+  # scopes
+  scope :needs_admin_response, -> { where('admin_status=? OR admin_status=?', 'applied', 'deferred') }
+  scope :has_admin_response, -> { where('admin_status!=? AND admin_status!=?', 'applied', 'deferred') }
+  scope :beginner_track, -> { where(track: 'beginner') }
+  scope :next_track, -> { where(track: 'next') }
+  scope :attending, -> { where(user_status: 'accepted') }
+
+  def self.allowed_params
+    [ :track, :over_18, :previous_attendance, :programming_experience, :reason,
+      :tshirt_size, :admin_status, :user_status, :gender, {dietary_requirements: []},
+      :extra_info]
+  end
+
+  def count_up_votes
+    @votes = Vote.all
+    @votes.select{|vote| vote.user_id == self.id && vote.vote_type == 'up'}.count
+  end
+
+  def count_down_votes
+    @votes = Vote.all
+    @votes.select{|vote| vote.user_id == self.id && vote.vote_type == 'down'}.count
+  end
+
+  def age
+    if over_18
+      'Over 18'
+    else
+      'Under 18'
+    end
+  end
+
+  def important_attributes
+    attributes = []
+    attributes << track if track == 'next'
+    attributes << gender if gender == 'Male'
+    attributes << age unless over_18
+    attributes
+  end
+
+  def send_application_thanks
+    UserMailer.application_received(self).deliver
```

Comment:
I'm not sure about your Mandrill mailer thing, but the Rails default mailer, you need to use `deliver_now` or `deliver_later` in Rails 4.2